src/packages/cli/package.json

```diff
@@ -95,10 +95,10 @@
     "lint-staged": "10.2.11",
     "log-update": "4.0.0",
     "make-dir": "3.1.0",
-    "mariadb": "2.4.1",
+    "mariadb": "2.4.2",
```

Comment:
What are we using this for?

Reply:
it's the library I'm using to execute MySQL and MariaDB SQL integration tests. It's actually great!!

Reply:
Oh it is - I was not aware we are interacting with MySQL directly - but it of course makes sense.